### ISSUE TITLE
✅ 🐞 Bug Report: In Mega Menu, inner containers break - [ED-10303]

### ISSUE THREAD

**Issue by Michaelgimii** [elementor/elementor#21813]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

If we add containers in the megamenu content and then set the desired width for them, the containers will be completely broken.

### Steps to reproduce

https://user-images.githubusercontent.com/88034902/228528443-aa37faa6-0f2b-4947-8a8a-10b848c29c58.mp4

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt


					
== Server Environment ==
	Operating System: WINNT
	Software: Apache/2.4.38 (Win32) OpenSSL/1.1.1a PHP/7.3.2
	MySQL version: mariadb.org binary distribution v5.6.42-84.2
	PHP Version: 7.3.2
	PHP Memory Limit: 128M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 64M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.1.1
	Site URL: http://localhost:80/elementor
	Home URL: http://localhost:80/elementor
	WP Multisite: No
	Max Upload Size: 64 MB
	Memory limit: 40M
	Max Memory limit: 256M
	Permalink Structure: /%year%/%monthnum%/%day%/%postname%/
	Language: en-US
	Timezone: Europe/berlin
	Admin Email: [REDACTED]
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.7.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/111.0.0.0 Safari/537.36

== Active Plugins ==
	Elementor
		Version: 3.12.0
		Author: Elementor.com

	Elementor Pro
		Version: 3.12.0
		Author: Elementor.com


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	


== Elementor Experiments ==
	Optimized DOM Output: Active
	Improved Asset Loading: Active
	Improved CSS Loading: Active
	Inline Font Icons: Inactive
	Accessibility Improvements: Inactive
	Additional Custom Breakpoints: Inactive
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Active
	Default to New Theme Builder: Inactive
	Hello Theme Header & Footer: Inactive
	Editor Top Bar: Active
	Landing Pages: Inactive
	Nested Elements: Active
	Lazy Load Background Images: Inactive by default
	Page Transitions: Inactive
	Notes: Inactive
	Loop: Active
	Scroll Snap: Inactive
	Menu: Active


== Log ==
	
Log: showing 20 of 322023-02-27 01:30:44 [info] elementor::elementor_updater Started 
2023-02-27 01:30:44 [info] Elementor/Upgrades - _on_each_version Start  
2023-02-27 01:30:44 [info] Elementor/Upgrades - _on_each_version Finished 
2023-02-27 01:30:44 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.11.1',
  'to' => '3.11.2',
)]
2023-03-20 01:11:35 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.11.2',
  'to' => '3.11.5',
)]
2023-03-20 01:11:36 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.2',
  'to' => '3.11.6',
)]
2023-03-20 01:11:36 [info] elementor-pro::elementor_pro_updater Started 
2023-03-20 01:11:36 [info] Elementor/Upgrades - _on_each_version Start  
2023-03-20 01:11:36 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-03-20 01:11:36 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-03-20 01:11:36 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.2',
  'to' => '3.11.6',
)]
2023-03-29 11:53:48 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.11.5',
  'to' => '3.12.0',
)]
2023-03-29 11:53:48 [info] elementor::elementor_updater Started 
2023-03-29 11:53:48 [info] Elementor/Upgrades - _on_each_version Start  
2023-03-29 11:53:48 [info] Elementor/Upgrades - _on_each_version Finished 
2023-03-29 11:53:48 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.11.5',
  'to' => '3.12.0',
)]
2023-03-29 14:37:27 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.6',
  'to' => '3.12.0',
)]
2023-03-29 14:37:27 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-03-29 14:37:27 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-03-29 14:37:27 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.6',
  'to' => '3.12.0',
)]

PHP: showing 4 of 4PHP: 2023-03-02 02:31:28 [error X 1][C:\xampp\htdocs\elementor\wp-content\plugins\elementor-pro\modules\theme-builder\classes\conditions-manager.php::295] Uncaught Error: Call to a member function update_meta() on bool in C:\xampp\htdocs\elementor\wp-content\plugins\elementor-pro\modules\theme-builder\classes\conditions-manager.php:295
Stack trace:
#0 C:\xampp\htdocs\elementor\wp-content\plugins\elementor-pro\modules\theme-builder\classes\conditions-manager.php(168): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->save_conditions(NULL, Array)
#1 [internal function]: ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->ajax_save_theme_template_conditions(Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#2 C:\xampp\htdocs\elementor\wp-content\plugins\elementor\core\common\modules\ajax\module.php(175): call_user_func(Array, Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#3 C:\xampp\htdocs\elementor\wp-includes\class-wp-hook.php(308): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request('')
#4 C:\xampp\htdocs\elementor\wp-includes\class-wp-hook.php(332): WP_Hook->apply_filters('', Array)
#5 C:\xampp\htdocs\elemen [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-03-11 14:20:00 [error X 4][C:\xampp\htdocs\elementor\wp-content\plugins\elementor\modules\safe-mode\module.php::11] Maximum execution time of 30 seconds exceeded [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-03-17 21:02:19 [notice X 1][C:\xampp\htdocs\elementor\wp-content\plugins\elementor-pro\modules\theme-builder\classes\conditions-manager.php::168] Undefined index: editor_post_id [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-03-29 14:38:36 [notice X 3][C:\xampp\htdocs\elementor\wp-content\plugins\elementor\core\common\modules\ajax\module.php::175] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 6 of 6JS: 2023-03-03 20:14:54 [error X 2][http://localhost:80/elementor/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.11.2:3:719506] Cannot read properties of undefined (reading 'widgets_settings') 
JS: 2023-03-04 12:28:29 [error X 10][http://localhost:80/elementor/wp-admin/load-scripts.php?c=1&loadchunk_0=underscore,jquery-core,jquery-migrate,backbone,jquery-ui-core,jquery-ui-mouse,wp-api-request,regenerator-runtime,wp-polyfill,wp-&loadchunk_1=hooks&ver=6.1.1:4:51824] Cannot read properties of undefined (reading 'replace') 
JS: 2023-03-09 11:53:29 [error X 3][http://localhost:80/elementor/wp-content/plugins/elementor/assets/js/editor-modules.min.js?ver=3.11.2:2:14150] An invalid argument supplied as multiple control value [array (
  'category' => 'InvalidElementData',
  'tag' => 'Element `undefined` got as `icon` value. Expected array or object.',
)]
JS: 2023-03-19 21:43:38 [error X 2][http://localhost:80/elementor/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.11.5:3:1087291] Cannot read properties of null (reading 'getBoundingClientRect') 
JS: 2023-03-29 10:08:23 [error X 2][http://localhost:80/elementor/wp-content/plugins/elementor-pro/assets/js/editor.min.js?ver=3.12.0:2:63837] Cannot read properties of undefined (reading 'createTemplate') 
JS: 2023-03-29 10:27:09 [error X 40][http://localhost:80/elementor/wp-content/plugins/elementor/assets/js/df16c9d56464dd4f0800.bundle.min.js:2:7826] Cannot read properties of null (reading 'dataset') 



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Incompatible

== Elementor Pro - Compatibility Tag ==

````

</details>

**Comment by nicholaszein** [comment]
Hey @Michaelgimii! 

Thank you for submitting your issue! 🙏 

We can confirm this is a bug, and we are working on a fix.

Best regards

**Comment by nicholaszein** [comment]
Hello @Michaelgimii!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro v3.14.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
